README.md

```diff
@@ -197,7 +197,7 @@ Example:
 ##### Download
 
 ```bash
-docker pull commercetools/commercetools-project-sync:3.11.0
+docker pull commercetools/commercetools-project-sync:4.0.0
```

Comment:
Where are release notes which justify major version change?

Reply:
@butenkor Release note is written when we draft the release in Github.

Reply:
@butenkor here is the draft of release note.

**Enhancements**
* Migrate java version from java 1.8 to java 11
* Update prerequisite section in README.md since JDK installation is no longer necessary.

**Dependency Updates**
* `com.commercetools:commercetools-sync-java`: `4.0.0` -> `4.0.1`
* `com.fasterxml.jackson.dataformat:jackson-dataformat-cbor` : `2.11.4` -> `2.12.2`
* `io.netty:netty-codec-http`: `4.1.59.Final` -> `4.1.63.Final`
* `com.diffplug.spotless`: `5.11.0`-> `5.11.1`
* `org.asynchttpclient:async-http-client`  `2.12.2` -> `2.12.3`